addons/winddeflection/stringtable.xml

```diff
@@ -1,4 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
+<<<<<<< HEAD
```

Comment:
the <<<<< ===== and >>>> lines are from a merge conflict warning and can be removed